std_msgs/Float64MultiArray act
---
std_msgs/Float64MultiArray obs
std_msgs/Float64 rew
std_msgs/Bool done